pulsar-broker/src/main/java/org/apache/pulsar/broker/admin/v2/Namespaces.java

```diff
@@ -267,6 +267,48 @@ public void revokePermissionOnSubscription(@PathParam("property") String propert
         internalRevokePermissionsOnSubscription(subscription, role);
     }
 
+    @PUT
```

Comment:
```suggestion
    @POST
```
Other API methods to grant permissions are using POST.

Reply:
I noticed the other grant endpoints use `POST`, but it seems like the wrong rest method because these grant operations are idempotent. I can switch to `POST` to be consistent, but I think it's more correct to use `PUT`.

Reply:
You are right about that. However, I think it's better to be consistent with the other methods which use POST for a similar purpose.

Reply:
Done.